gpu-operator/release-notes.rst

```diff
@@ -31,8 +31,134 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
+  Refer to the :ref:`GPU Operator Component Matrix`
+  on the platform support page.
+
+* Added support for the following software component versions:
+
+    - NVIDIA Container Toolkit version v1.15.0
+    - NVIDIA MIG Manager version v0.6.0
+    - NVIDIA Driver Manager for K8s v0.6.8
+    - NVIDIA Kubernetes Device Plugin v0.15.0
+    - DCGM 3.3.3-1
+    - DCGM Exporter 3.3.5-3.4.0
+    - Node Feature Discovery v0.15.4
+    - NVIDIA GPU Feature Discovery for Kubernetes v0.15.0
+    - NVIDIA MIG Manager for Kubernetes v0.6.0
+    - NVIDIA KubeVirt GPU Device Plugin v1.2.6
+    - NVIDIA vGPU Device Manager v0.2.5
+
+* Added support for Kubernetes v1.29 and v1.30.
+  Refer to :ref:`Supported Operating Systems and Kubernetes Platforms`.
+
+* Added support for NVIDIA GH200 Grace Hopper Superchip as a generally available feature.
+  Refer to :ref:`supported nvidia gpus and systems`.
+
+  The following prerequisites are required for using the Operator with GH200:
+
+  - Run Ubuntu 22.04, the 550.54.15 GPU driver, and an NVIDIA Linux kernel, such as one provided with a ``linux-nvidia-<x.x>`` package.
+  - Add ``init_on_alloc=0`` and ``memhp_default_state=online_movable`` as Linux kernel boot parameters.
+  - Run the NVIDIA Open GPU Kernel module driver.
+
+* Removed support for Kubernetes pod security policy (PSP).
+  PSP was deprecated in the Kubernetes v1.21 release and removed in v1.25.
+
+.. _v24.3.0-fixed-issues:
+
+Fixed Issues
+------------
+
+* Installation on Red Hat OpenShift Container Platform 4.15 no longer requires a workaround related to
+  secrets and storage for the integrated image registry.
+* Previously, the vGPU Device Manager encountered an error if no NVIDIA devices were found in ``/sys/class/mdev_bus``.
+* Previously, the MOFED validation init container would run for the GPU driver pod.
+  In this release, the init container no longer runs because the MOFED installation check is performed by the Kubernetes Driver Manager init container.
+* Previously, for Red Hat OpenShift Container Platform, the GPU driver installation would fail when the Linux kernel version did not match the ``/etc/os-release`` file.
+  In this release, the Kernel version is determined from the running kernel to prevent the issue.
+  Refer to Github `issue #617 <https://github.com/NVIDIA/gpu-operator/issues/617>`__ for more details.
+* Previously, if the metrics for DCGM Exporter were configured in a config map and the cluster policy
+  specified the name of the config map as ``<namespace>:<config-map>`` in the ``DCGM_EXPORTER_CONFIGMAP_DATA`` environment variable, the exporter
+  pods could not read the configuration from the config map.
+  In this release, the role used by the exporter is granted access to read from config maps.
+* Previously, the GPU Driver container on a Ubuntu 22.04 node supported the 5.15 LTS Linux kernel only.
+  In this release, Ubuntu 22.04 is supported with 5.15, 5.19, 6.2, and 6.5.
```

Comment:
Actually, we should not call out support for the 6.x kernels with GPU Operator 24.3.0 since the driver container fix for the gcc issues has not made it into an official driver container release.

Apologies for the confusion here, we should not have kept the JIRA associated with this in the 24.3.0 initiative. Let's rollback the docs changes you made in e02077c1c776e081b8ae7bf086cdcad843948d88